src/pages/LoadUnload/components/MapSettingsDisplay.tsx

```diff
@@ -49,14 +50,15 @@ export const LoadUnloadMapSettingsDisplay = ({
   setShowDateTimeModal,
   ...otherProps
 }: MapSettingsDisplayProps) => {
+  const { t } = useTranslation()
   const { address, dateTime } = useLoadUnloadPageContext()
   const showAddressForm = () => setShowAddressForm(true)
   const showDateTimeModal = () => setShowDateTimeModal(true)
 
   return (
     <MapPanelContent data-testid="map-settings" {...otherProps}>
       <Tabs label="Voer adres en moment in of bekijk dataverantwoording">
```

Comment:
We should also translate the label right?

Reply:
Yeah, I'll add that.